src/python/pants/build_graph/target.py

```diff
@@ -151,13 +144,6 @@ def is_scala(self):
     """Returns True if the target has scala sources."""
     return self.has_label('scala')
 
-  # DEPRECATED to be removed after 0.0.29
-  # do not use this method, use an isinstance check on a yet-to-be-defined mixin
```

Comment:
Thoughts on the mixin idea?

Reply:
I think that was a throwaway idea. There's almost nowhere were we care about something being a tests target of any type, rather than specifically a PythonTests or JunitTests etc. So type tests seem appropriate for now.